packages/cozy-flags/src/index.js

```diff
@@ -1,8 +1,14 @@
+/* global __ENABLED_FLAGS__ */
+
 import isNode from 'detect-node'
 export { default as FlagSwitcher } from './browser/FlagSwitcher'
 
 const flag = isNode
   ? require('./node/flag').default
   : require('./browser/flag').default
 
+if (__ENABLED_FLAGS__ && Array.isArray(__ENABLED_FLAGS__)) {
```

Comment:
```suggestion
if (typeof __ENABLED_FLAGS__ !== 'undefined' && Array.isArray(__ENABLED_FLAGS__))
```

mmh, this will throw a `ReferenceError` if `__ENABLED_FLAGS__` is not defined, `typeof` can be used here.